### DISCUSSION TITLE
Fulfillment Inbound: Shipment Related Dates

### DISCUSSION THREAD

**Discussion by ProNotion** [amzn/selling-partner-api-models#3429] [General] [answered]
In Seller Central we see the Shipping Queue and each shipment has a created date/time and last updated date/time. Where in the API can I find these values as they do not appear to be available as a part of the [InboundShipmentInfo](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-reference#inboundshipmentinfo)?

**Comment by ProNotion** [comment]
This is really quite an oversight in my opinion because inbound shipments without any sort of date reference are just not very useful at all. There must be somewhere in the API that this information can be found without having to resort to going into Seller Central?

**Reply by shenglingshi** [reply]
I have the same  question , why so long time not answer? @amazon-auto

**Comment by GJ-JG** [comment]
Hi @ProNotion , you can try specify ```lastUpdatedAfter``` and ```lastUpdatedBefore``` in the API request of getShipments with the value given by users.

**Reply by ProNotion** [reply]
That still doesn't give you the date the shipment was created

**Comment by siriustigtr** [comment]
Same problem , I tried both v0 and new endpoint and not able to get Created and LastUpdated Date.

**Comment by puppsupr** [comment]
Hi,
Please look at the new FBA Inbound [listInboundPlans](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#listinboundplans) operation using which you can get inbound plans based on status and sort order.

**Reply by ProNotion** [reply]
@puppsupr  The new FBA Inbound API only allows retrieval of current _plans_ and not inbound shipments that are already shipped? In addition, it will only provide the actual plan creation/update dates and not those of the shipments themselves? It seems to have been unnecessarily difficult to get at this critical data for our shipments?

**Reply by puppsupr** [reply, marked as answer]
Hi @ProNotion,
You filter [listInboundPlans](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#listinboundplans) using [status](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#status) field to get SHIPPED inbounds. Yes the API is at the inbound plan level. If you want shipment level data you can continue using [getShipments](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-reference#getshipments) v0 operation as this will not be deprecated.
